pkg/olm/installplan.go

```diff
@@ -0,0 +1,163 @@
+package olm
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Builder provides a struct for installplan object from the cluster and a installplan definition.
+type Builder struct {
```

Comment:
We have multiple elements under olm package. Please re-name Builder to InstallPlanBuilder

Reply:
yea, sorry it was developed as a separate pkg, and then we moved it ti the olm